board/safety/safety_volkswagen.h

```diff
@@ -1,18 +1,20 @@
-const int VW_MAX_STEER = 250;               // 2.5 nm
-const int VW_MAX_RT_DELTA = 75;             // 4 max rate up * 50Hz send rate * 250000 RT interval / 1000000 = 125 ; 125 * 1.5 for safety pad = 75
-const uint32_t VW_RT_INTERVAL = 250000;     // 250ms between real time checks
-const int VW_MAX_RATE_UP = 4;               // 5.0 nm/s available rate of change from the steering rack
-const int VW_MAX_RATE_DOWN = 10;            // 5.0 nm/s available rate of change from the steering rack
-const int VW_DRIVER_TORQUE_ALLOWANCE = 80;
-const int VW_DRIVER_TORQUE_FACTOR = 3;
-
-struct sample_t vw_torque_driver;           // last few driver torques measured
-int vw_rt_torque_last = 0;
-int vw_desired_torque_last = 0;
-uint32_t vw_ts_last = 0;
+const int VOLKSWAGEN_MAX_STEER = 250;               // 2.5 nm
+const int VOLKSWAGEN_MAX_RT_DELTA = 75;             // 4 max rate up * 50Hz send rate * 250000 RT interval / 1000000 = 125 ; 125 * 1.5 for safety pad = 75
+const uint32_t VOLKSWAGEN_RT_INTERVAL = 250000;     // 250ms between real time checks
+const int VOLKSWAGEN_MAX_RATE_UP = 4;               // 5.0 nm/s available rate of change from the steering rack
```

Comment:
Proposed comment touchup: // 2.0 nm/s rate of change (EPS side delta-limit of 5.0nm/s)
Fixes a factually incorrect/outdated figure